proto/controller.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package controller;
+
+option go_package = "github.com/google/kne/proto/controller";
+
+// Topology manager service definition.
+service TopologyManager {
+  rpc CreateTopology (CreateTopologyRequest) returns(CreateTopologyResponse) {}
+}
+
+// Request message to create a topology from a file.
+message CreateTopologyRequest {
+  string file_name = 1;
```

Comment:
how is that going to work... the cli will have the file locally how are you getting the file onto the service running in k8s?

Reply:
Changed CreateTopologyRequest to take in topology proto